lib/ansible/plugins/connection/nspawn.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import (absolute_import, division, print_function)
```

Comment:
There needs to be a copyright and license header above this.  All controller-side code in ansible is GPLv3+.  You can take a look at another connection plugin (for instance, chroot.py https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/connection/chroot.py ) for an example.